static/src/javascripts/projects/common/modules/ui/sticky.js

```diff
@@ -1,64 +1,83 @@
 define([
     'bonzo',
-    'common/utils/$',
     'common/utils/config',
     'common/utils/mediator',
     'fastdom',
-    'lodash/objects/defaults',
-    'lodash/functions/bindAll'
+    'lodash/objects/defaults'
 ], function (
     bonzo,
-    $,
     config,
     mediator,
     fastdom,
-    defaults,
-    bindAll) {
+    defaults) {
+
+    var header = config.switches.viewability
```

Comment:
I would've hoped so, but this is copied straight-out from enhanced/common.js. I'll see if this could be done earlier during the bootstrap process.